recipes/default.rb

```diff
@@ -9,7 +9,7 @@
 poise_service_user node['consul']['service_user'] do
   group node['consul']['service_group']
   shell node['consul']['service_shell'] unless node['consul']['service_shell'].nil?
-  not_if { node.platform_family?('windows') }
```

Comment:
`platform_family?` method is not available in Chef <= 12.6.0 (https://github.com/johnbellone/consul-cookbook/pull/417).
Let's just keep it as is and add `FC028` to the ignore list in `.foodcritic` file.